### DISCUSSION TITLE
recovering/checking mastodon account

### DISCUSSION THREAD

**Discussion by dwalkerart** [mastodon/mastodon#26365] [Q&A]
Hello,
I am in need of help with searching or if possible, recovering my accounts for mastodon.social and mastodon.disabled. I've tried to see if the confirmation email was in my inbox/spam but to no avail and can't seem to get a email when I tried the forgot password as well. 

It wasn't long ago when I made one and I want to see if it's still there somewhere hopefully as I am trying to get it. :(

**Comment by renchap** [comment]
You can reach out to our support team on staff@mastodon.social for this, we here on Github cant help you on this topic.
